poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
```

Comment:
is there a reason why the lock file has to change almost completely? There are internal libraries that have been downgraded (e.g. `alibi` and `alibi-detect`) that at least should be reverted.